### PR TITLE
player movement with wasd. acceleration and velocity working

### PR DESCRIPTION
**Pull request by shraymittal**

### CODE REVIEW

**Comment by Noviv** [comment]
will the controller movement be a separate PR? also this doesn't seem to allow for easy switching between the two options (like if Elmer or anyone else wants to test the other mode)

**Comment by shraymittal** [comment]
We can leave this open while I add controller support. I put this here mostly if you guys wanted/needed it today. I'll have controller pushed to this PR tomorrow

File: BismarkUnchained/scenes/KinematicBody2D.gd [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+	var accel = 30
+	if diff_velocity.length() >= 10 || velocity.length() == 0:
+		accel = 60
+	var new_velocity = curr_velocity + diff_velocity.normalized() * accel * delta
+	if new_velocity.length() > 10:
+		new_velocity = velocity
+	if new_velocity.length() < 0.5 && velocity.length() == 0:
+		new_velocity = velocity
+	velocity = new_velocity
+	move_and_collide(velocity)
```
Comment (Noviv) [inline BismarkUnchained/scenes/KinematicBody2D.gd]: multiply by `delta`

File: BismarkUnchained/scenes/KinematicBody2D.gd [hunk 2]
```diff
@@ -1,5 +1,29 @@
 extends KinematicBody2D
 
-func _physics_process(delta):
-	move_and_collide(Vector2(50, 0))
+var velocity
```
Comment (Noviv) [inline BismarkUnchained/scenes/KinematicBody2D.gd]: can be moved into `_process`, depending on what you're doing for controller input

**Review by Noviv — CHANGES_REQUESTED** [review]

**Comment by Noviv** [comment]
Also, this PR needs to somehow start updating the `time_delta` global variable, so we can start working towards our core feature.

**Review by Noviv — APPROVED** [review]

**Pull request merged**